lms/static/js/discovery/views/course_card.js

```diff
@@ -33,10 +33,17 @@
                 var data = _.clone(this.model.attributes);
                 var userLanguage = '',
                     userTimezone = '';
+                var currentDate = new Date();
                 if (this.model.userPreferences !== undefined) {
                     userLanguage = this.model.userPreferences.userLanguage;
                     userTimezone = this.model.userPreferences.userTimezone;
                 }
+                var afterEnrollStarted = (currentDate < new Date(data.start)) ? "The enrollment is open" : "The course is on";
```

Comment:
Are we sure that enrollment **always** closes once a course starts?

Reply:
Also, the var's naming `afterEnrollStarted` and its value seems a bit unclear to me...

Is there any way to separate start / enroll logic?

Reply:
Reworked, Thanks!

Reply:
Approved, although I'm not sure I understand the business logic well (still confused by enrolling start date VS course start date logic).